Articles/Blog/HelloHeatEquation.md

```diff
@@ -1,143 +1,99 @@
-
-[FD1D\_HEAT\_EXPLICIT](https://people.sc.fsu.edu/~jburkardt/c_src/fd1d_heat_explicit/fd1d_heat_explicit.html "FD1D\_HEAT\_EXPLICIT") is a simple 1-dimensional Heat Equation solver which uses the explicit version of the finite difference method to handle time integration. Versions of this library are written in C, C++, Fortran90, Matlab, and Python. A [code example from the library](https://github.com/betterscientificsoftware/hello-heat-equation/blob/master/LibraryImplementations.md "FD1D") demonstrates a sequential implementation, but can easily be adapted to become parallelized.
-
-While this code is almost identical the hand-coded example above, this library does include some extra features that allows users to implement some additional functionality including writing results to different kinds of R8 files, adding a timestamp, and solving for the Courant-Friedrichs-Loewy coefficient.
-
-### More Implementations!
-Using the Heat Equation to demonstrate the power of a programming model or mathematical library is not new. In fact, it is common tutorial exercise for new scientific programmers. Since there are too many interesting examples to include in this article, we've started [a repository showcase](https://github.com/betterscientificsoftware/hello-heat-equation) to collect the many examples.
+## Why this form of the Heat Equation?
+While the one dimensional heat equation might not be sufficient to
+model solar heating of the Earth, even in this relatively simple form,
+interesting and meaningful science questions may be investigated involving
+both transient and steady-state behavior. Examples are determining if water
+pipes embedded in log cabin (solid wood) walls would freeze during a cold
+storm or the required thickness of insulation around fuel pipeline to avoid
+combustion from a fast burning brush fire. These and many other questions
+involving heat flow can be investigated with this relatively simply form of
+the heat equation.
+
+Solutions to the one dimensional heat equation are also very easy for
+beginners to visualize. This can help them understand how different
+approaches work. The simplicity of the physics and mathematics also
+allows beginners to focus more on the implementation details; the
+programming language, parallel execution paradigm, discretization and
+mathematical support libraries.
+
+## A Hello CSE World Repository
+We are announcing the beginnings of an effort to populate and maintain a
+repository of implementations of the heat equation demonstrating various
+aspects of CSE software. The resulting repository will be useful in training
+about fundamental aspects of the design and development of CSE software as
+well as provide examples of continuous integration testing including
+demonstrating the use of ECP-CI resources.
+
+Each implementation we gather into this repository will have to conform
+to a *minimal* set of requirements including providing a fully self-contained
```

Comment:
We are not asking the examples to be runnable, only that they use common naming scheme for variables.

Reply:
As per side-bar conversation at DOE Data Day, we agreed to keep snipits for now in the repo created for that purpose. Eventually, those will be replaced with a repo hosting fully working examples of same or similar lineage.